test/restore.bats

```diff
@@ -324,3 +324,92 @@ function teardown() {
 	output=$(crictl inspect -o table "$ctr_id" | grep ^State)
 	[[ "${output}" == "${ctr_status_info}" ]]
 }
+
+@test "crio restore upon entering KUBENSMNT" {
+	start_crio
+	pod_id=$(crictl runp "$TESTDATA"/sandbox_config.json)
+	pod_list_info=$(crictl pods --quiet --id "$pod_id")
+
+	output=$(crictl inspectp -o json "$pod_id")
+	[[ "$output" != "" ]]
+	pod_status_info=$(echo "$output" | jq ".status.state")
+	pod_ip=$(echo "$output" | jq ".status.ip")
+	pod_created_at=$(echo "$output" | jq ".status.createdAt")
```

Comment:
would be nice to use here-file for this, but since other places doesn't use it, never mind.

Reply:
I'm going to go make changes anyway, so why not :)